.github/check_license.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+
+usage() {
+  echo "$0 usage: [-r n] license file"
+  echo ""
+  echo "where:"
+  echo "-r n  max line number (1-based) where the first line of license header should start."
+  echo "      First line of the header must fall within range [1,n] (default: 1)"
+  echo ""
+  echo "SPECIAL TAGS: they can be used inside files to control behavior of this tool"
+  echo ""
+  echo "dlaf-no-check-license       skip any check and return 0"
+  echo "dlaf-multi-license-check    check license in the entire file (-r option is ignored)"
```

Comment:
Multi license check just ignores the range parameter, i.e. if a file has the tag in it, the license can be wherever in it.